warehouse/models/intermediate/gtfs_quality/guidelines_checks/int_gtfs_quality__trip_id_alignment.sql

```diff
@@ -1,30 +1,18 @@
-WITH feed_guideline_index AS (
-    SELECT * FROM {{ ref('int_gtfs_quality__rt_feed_guideline_index') }}
-),
-
-fct_daily_rt_feed_files AS (
-    SELECT * FROM {{ ref('fct_daily_rt_feed_files') }}
-),
-
-fct_daily_rt_feed_validation_notices AS (
-    SELECT * FROM {{ ref('fct_daily_rt_feed_validation_notices') }}
-),
+{% set min_date_array = dbt_utils.get_column_values(table=ref('fct_daily_rt_feed_validation_notices'), column='date', order_by = 'date', max_records = 1) %}
+{% set first_check_date = min_date_array[0] %}
```

Comment:
nit: I don't love doing this at compile-time... do you have thoughts on replacing this with a `when not exists ...` in the case statement?

Reply:
Discussed offline, I converted these to separate CTEs